question28/js/password.js

```diff
@@ -0,0 +1,35 @@
+import { validateInputValue } from "./modules/validations";
+
+const submitButton = document.getElementById('js-submit-button');
+const passwordInputArea = document.querySelector(".js-password");
+const confirmPasswordInputArea = document.querySelector(".js-confirm-password");
+
+
+const checkAllValidity = () => {
+  return document.getElementsByTagName("input").length === document.getElementsByClassName("valid").length && passwordInputArea.value === confirmPasswordInputArea.value;
+}
```

Comment:
[ IMO ]
Since it returns a boolean value, we felt that the function name is** would be easier to understand.

Reply:
Thank you for your review!
You are right. I changed the variable names to ``isValidAllInputsValue`` from ``checkAllValidity``. What do you think? Please advise if there is a more appropriate one. :bow:

fixed commit : https://github.com/csakyo/homework/pull/43/commits/c1ac6abb0927d113efa4d7d37940ba7df8ab5256